File: app/src/main/java/com/example/android/miniweather/CityWeather.java
package com.example.android.miniweather;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CityWeather {

    @SerializedName("location")
    @Expose
    private Location location;
    @SerializedName("forecast")
    @Expose
    private Forecast forecast;

    public Location getLocation() {
        return location;
    }

    public void setLocation(Location location) {
        this.location = location;
    }

    public Forecast getForecast() {
        return forecast;
    }

    public void setForecast(Forecast forecast) {
        this.forecast = forecast;
    }

}

File: app/src/main/java/com/example/android/miniweather/Models/CityWeather.kt
package com.example.android.miniweather.Models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class CityWeather(@SerializedName("location") @Expose val location: Location,
    @SerializedName("forecast") @Expose val forecast: Forecast)